resources/css/global.css

```diff
@@ -13,6 +13,11 @@ h2 {
   margin: 0 auto 0 auto;
 }
 
+/*All purpose yellow background*/
+.bg-yellow {
+  background-color: #FFC82B;
+}
+
```

Comment:
you can probably get away without using bg-yellow and use tailwind's bg-yellow-<shade> helpers. ex: `bg-yellow-500`